src/views/SystemIntake/ContactDetails/GovernanceTeamOptions.tsx

```diff
@@ -19,7 +19,12 @@ const GovernanceTeamOptions = ({ formikProps }: GovernanceTeamOptionsProps) => {
   return (
     <FieldArray name="governanceTeams.teams">
       {arrayHelpers => (
-        <>
+        <fieldset className="margin-0 padding-0 border-0">
+          <legend className="usa-sr-only">
+            For the checkboxes below, select all the teams you&apos;ve
```

Comment:
Is there a reason we're escaping the apostrophe here using `&apos;`? I'm curious how that reads vs the literal `'`.

Reply:
It comes from: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-unescaped-entities.md

I have no real opinion over it. We can disable if if we want.

Reply:
I created https://jiraent.cms.gov/browse/EASI-840 as a placeholder ticket.

I'm going to merge this ticket as is now because we have these HTML entities throughout the application and fixing it in this once place wont fix our application wide implementation.